server/neptune/workflows/internal/deploy/revision/queue/worker_test.go

```diff
@@ -269,74 +282,121 @@ func TestWorker_CompareCommit_SkipDeploy(t *testing.T) {
 
 }
 
-func TestWorker_CompareCommit_Deploy(t *testing.T) {
+func TestWorker_CompareCommit_DeployAhead(t *testing.T) {
```

Comment:
Any reason you didn't want to just move these tests to a `revision_processor_test` file? Seems annoying to have to continously test with the queue worker async dynamic.

Reply:
Will move them